src/chatlogpiece.cpp

```diff
@@ -5,27 +5,31 @@ chatlogpiece::chatlogpiece()
   name = tr("UNKNOWN");
   showname = tr("UNKNOWN");
   message = tr("UNKNOWN");
+  color = 0;
   p_is_song = false;
   datetime = QDateTime::currentDateTime().toUTC();
 }
 
 chatlogpiece::chatlogpiece(QString p_name, QString p_showname,
-                           QString p_message, bool p_song)
+                           QString p_message, bool p_song, int p_color)
 {
   name = p_name;
   showname = p_showname;
   message = p_message;
   p_is_song = p_song;
+  color = p_color;
   datetime = QDateTime::currentDateTime().toUTC();
 }
 
 chatlogpiece::chatlogpiece(QString p_name, QString p_showname,
-                           QString p_message, bool p_song, QDateTime p_datetime)
+                           QString p_message, bool p_song, int p_color,
+                           QDateTime p_datetime)
```

Comment:
p_color and p_datetime were re-ordered, needs to be double-checked that it doesn't affect anything

Reply:
uh, argument order on declaration and definition is consistent.